microsoft-365/security/defender-endpoint/get-file-statistics.md

```diff
@@ -39,6 +39,7 @@ Retrieves the statistics for the given file.
 ## Limitations
 
 1. Rate limitations for this API are 100 calls per minute and 1500 calls per hour.
+2. Maximum Value for Lookbackhours is 720 Hours(30days).
```

Comment:
```suggestion
2. Maximum value for Lookback hours is 720 hours(30 days).
```

Reply:
lookBackHours is the exact match of the query. So this change is not needed.

Reply:
@msbemba, thank you for clarifying.